sample-geovelocity/siddhi-execution-geovelocity/src/main/java/org/wso2/extension/siddhi/execution/geovelocity/api/GeoVelocityData.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.wso2.extension.siddhi.execution.geovelocity.api;
+
+/**
+ * This class is for Geo-velocity data.
+ */
+public class GeoVelocityData {
+
+    private Long lastLoginTime;
+    private Integer ristrictedLocationCombination;
+
+    public GeoVelocityData(Long lastLoginTime, Integer ristrictedLocationCombination) {
+
+        this.lastLoginTime = lastLoginTime != null ? lastLoginTime : 0L;
+        this.ristrictedLocationCombination = ristrictedLocationCombination != null ? ristrictedLocationCombination : 0;
+    }
+
+    public Long getLoginBehaviourBasedRisk() {
+        return lastLoginTime;
```

Comment:
Keep a new line after the method declaration

Reply:
Fixed in https://github.com/wso2/samples-is/pull/265